ycmd/tests/javascript/testdata/jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is the config filename supposed to be `jsconfig.json` or `tsconfig.json`?

Reply:
Both are supported. The only difference is that `allowJs` must be set to `true` in `tsconfig.json` for JavaScript projects:
```json
{
  "compilerOptions": {
    "allowJs": true
  }
}
```

Reply:
👍